test/toFixed.spec.js

```diff
@@ -10,6 +10,13 @@ tape('toFixed', function (test) {
   var template = Handlebars.compile('{{toFixed number}}');
   var expected = '1.00';
   var actual = template({ number: 1 });
-  test.plan(1);
+  test.plan(2);
   test.equal(actual, expected, 'Works');
+  test.throws(
+    function () {
+      template({ number: 'abc123' })
+    },
+    /number\-like value\.$/,
+    'Errors when passed an unparseable value'
```

Comment:
Yeah, I saw a similar example in the MDN docs. Perhaps I'll tweak the test to have no numeric characters in that assertion.